src/pages/place.tsx

```diff
@@ -61,26 +82,25 @@ export default function PlacePage() {
         share({
           title: place.title || undefined,
           text: `${l("general.place_share")}${shareableText}`,
-          url: location.origin + locations.place(place.id),
+          url: location.origin + locations.place(place.base_position),
           thumbnail: place.image || undefined,
         })
-        track(SegmentPlace.Share, {
-          placeId: place.id,
-        })
       }
     },
     [place, track]
   )
 
-  const loading = accountState.loading
-
   const [ff] = useFeatureFlagContext()
 
   if (ff.flags[FeatureFlags.Maintenance]) {
     return <MaintenancePage />
   }
 
-  if (!loading && !place) {
+  if (
+    placeRetrivedState.version > 0 &&
```

Comment:
we should consider adding a new prop like `firstLoading` in `decentraland-gatsby`

Reply:
👍